e2e/vertical-full-page-map.spec.ts

```diff
@@ -0,0 +1,118 @@
+import { test, expect } from '@playwright/test';
+
+test.describe('full page map test suite', () => {
+  test.beforeEach(async ({page}) => {
+    await page.goto('http://localhost:5042/locations_full_page_map');
+    await page.getByPlaceholder('Search for locations').click();
+  });
+
+  test('can search and get results', async ({ page }) => {
+    await page.getByPlaceholder('Search for locations').fill('virginia');
+    await page.getByPlaceholder('Search for locations').press('Enter');
+    const count = await page.locator('#js-answersVerticalResults').count();
+    expect(count).toBeGreaterThan(0);
```

Comment:
even if the `virginia` search didn't happen, the count would be greater than 0 because of the search conducted when first visiting the page. instead, can we check that the count is different from before the search vs. after?